etna/lib/etna/clients/metis/workflows/metis_upload_workflow.rb

```diff
@@ -110,8 +110,6 @@ class Upload
 
           def initialize(source_file: nil, next_blob_size: nil, current_byte_position: nil)
             self.source_file = source_file
-            self.next_blob_size = next_blob_size
```

Comment:
is there a reason for this deletion?

Reply:
It seems redundant with the subsequent two lines of code?

Reply:
oh, hmmm, I see.